workspace/build_manifest.go

```diff
@@ -59,41 +58,53 @@ type ExecPhase struct {
 
 func (e *ExecPhase) EnvironmentVariables(ctx context.Context, envName string, data runtime.RuntimeEnvironmentData) []string {
```

Comment:
Seems like this function should now return an error because it can fail.

Reply:
Makes some sense